appserver/monitoring-console/core/src/main/java/fish/payara/monitoring/store/InMemoryAlarmService.java

```diff
@@ -108,11 +108,14 @@
     public void init() {
         isDas = serverEnv.isDas();
         changedConfig(parseBoolean(serverConfig.getMonitoringService().getMonitoringEnabled()));
-        Watch watch = new Watch("Metric Collection Duration", new Metric(new Series("ns:monitoring CollectionDuration")))
-                .red(800L, 3, true, 800L, 5, false)
-                .amber(600L, 3, true, 600L, 5, false)
-                .green(-400L, 1, false, null, null, false);
-        addWatch(watch);
+        addWatch(new Watch("Metric Collection Duration", new Metric(new Series("ns:monitoring CollectionDuration")))
```

Comment:
NB. The watch for a metric provided by `InMemoryMonitoringDataRepository` had to be added here as this would otherwise cause a cyclic dependency between the two.